lib/siilar/client.rb

```diff
@@ -75,6 +74,10 @@ def base_options
         }
       }
 
+      if requests_timeout
+        options.merge!(timeout: requests_timeout)
```

Comment:
I don't know very well how their API perform IRL, so I can't say.
Let's keep it that way for now :+1: